core/Url.php

```diff
@@ -916,4 +916,18 @@ public static function addCampaignParametersToMatomoLink(
         $pathAndQueryString = UrlHelper::getPathAndQueryFromUrl($url, $newParams, true);
         return 'https://' . $domain . '/' . $pathAndQueryString;
     }
+
+    /**
+     * Create an external link tag with optional campaign params if link goes to matomo.org
+     */
+    public static function getExternalLink(
```

Comment:
Maybe a naming like `getExternalLinkHtmlMarkup`, `getExternalLinkTag` or similar would be more meaningful. Currently the name of the method doesn't really imply what exactly it returns.

Reply:
Yeah, I was thinking about that, wanted to keep it shorter. I'll go with `getExternalLinkTag` as it's not a full markup for the whole link, it's just the opening tag with its attributes.